modules/ocf_apphost/files/vhost-app.jinja

```diff
@@ -0,0 +1,38 @@
+# {{vhost.comment}}
+server {
+    listen {{vhost.port}};
+    listen [::]:{{vhost.port}};
+    server_name "{{vhost.fqdn}}";
+
+    # Proxy /.well-known/ to www.o.b.e/.well-known/ so death can request
+    # Let's Encrypt certificates for app vhosts
```

Comment:
this is kinda weird. imo servers should request their own certificates (this is what the RFC i sent out suggested)

why centralize everything on death? what if we have servers that want to use LE but don't mount NFS?